contrib/completion/zsh/_docker

```diff
@@ -2316,14 +2316,28 @@ __docker_caching_policy() {
 
 __docker_commands() {
     local cache_policy
+    integer force_invalidation=0
 
     zstyle -s ":completion:${curcontext}:" cache-policy cache_policy
     if [[ -z "$cache_policy" ]]; then
         zstyle ":completion:${curcontext}:" cache-policy __docker_caching_policy
     fi
 
-    if ( [[ ${+_docker_subcommands} -eq 0 ]] || _cache_invalid docker_subcommands) \
-        && ! _retrieve_cache docker_subcommands;
+    if ( [[ ${+_docker_hide_legacy_commands} -eq 0 ]] || _cache_invalid docker_hide_legacy_commands ) \
```

Comment:
In replacement of `[[ ${+_docker_hide_legacy_commands} -eq 0 ]]` ? I'm still not sure of the real meaning of `${+`.